haiku-misc/gadgeteer/gadgeteer-0.8.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Command line for creating GUI forms"
+DESCRIPTION="Gadgeteer is a command line app for creating small GUI forms \
+that users can fill out or play with. Once the users press the \"DONE\" \
+button at the bottom of the screen, the command returns the user's input \
+values so they could be used for different purposes."
+HOMEPAGE="https://www.goodeveca.net/beos/"
+COPYRIGHT="2016 Pete Goodeve
+	2006 Kirill Simonov"
+LICENSE="MIT"
```

Comment:
Please also add `Unknown` as a license. I've investigated the source code and found that Gadgeteer doesn't have a license, only the bundled `libYAML` have one.

Reply:
Pete can easily be contacted to help sorting that out.

Reply:
I've sent a message to Pete on the Haiku forums.